deploy/sdk/src/dynamo/sdk/tests/pipeline.py

```diff
@@ -160,7 +156,7 @@ class Frontend:
     def __init__(self) -> None:
         print("Starting frontend")
 
-    @dynamo_endpoint(is_api=True)
+    @dynamo_endpoint(transports=[DynamoTransport.HTTP])
```

Comment:
You'll need to modify service.py (https://github.com/ai-dynamo/dynamo/pull/960/files#diff-c836b61173ca088f6457f4cafabbdf9cedd4ca64b81545631f50622a909f4f06) to now check for `transports` including `DynamoTransport.HTTP` instead of the current check for `is_api=True`